libfoundation/include/foundation.h

```diff
@@ -2723,6 +2723,19 @@ MC_DLLEXPORT bool MCStringNormalizedCopyNFKD(MCStringRef, MCStringRef&);
 MC_DLLEXPORT bool MCStringSetNumericValue(MCStringRef self, double p_value);
 MC_DLLEXPORT bool MCStringGetNumericValue(MCStringRef self, double &r_value);
 
+enum MCStringLineEndingStyle
+{
+    kMCStringLineEndingStyleLF,
+    kMCStringLineEndingStyleCR,
+    kMCStringLineEndingStyleCRLF
+};
+
+MC_DLLEXPORT bool MCStringNormalizeLineEndings(
+                            MCStringRef p_input, 
+                            MCStringLineEndingStyle p_to_style, 
+                            MCStringRef& r_output, 
```

Comment:
I think this function actually (annoyingly) needs another parameter `bool p_normalize_ps_ls`. We want to do this when importing text into LiveCode, but not when exporting outside of LiveCode (otherwise you can't construct strings which contains PS / LS to export without having to do everything yourself - it's an edge case, but it is how the previous functions worked).